src/sass/post-911-gib-status.scss

```diff
@@ -82,11 +82,27 @@
     }
 
     @media print {
+      font: $font-serif;
+      color: $color-gray-dark;
       padding: 0;
+      a {
+        text-decoration: none;
+        &:after {
+          content:" (https://vets.gov" attr(href) ")";
+        }
+      }
+
+      button {
+        display: none;
+      }
     }
   }
 }
 
+.no-print-no-sr {
+    display: none !important;
```

Comment:
This feels like it'd probably be helpful outside of GIBS. What do you think of moving it to the main stylesheet?

Reply:
Yeah, I went back and forth on making it a mixin, but applying directly to the va stylesheet is probably more appropriate.